api/arc.go

```diff
@@ -190,6 +190,51 @@ type ErrorMalformed struct {
 	Type interface{} `json:"type"`
 }
 
+// ErrorNotFound defines model for ErrorNotFound.
+type ErrorNotFound struct {
```

Comment:
you can reuse fields from `ErrorGeneric` in `ErrorNotFound` and others simply by embedding to avoid duplicating exact same set of fields:

```
type ErrNotFound {
   ErrorGeneric 
}

type ErrorOutputs struct {
  ErrorGeneric 
}
```

or by type aliases:

```
type ErrorNotFound ErrorGeneric 
type ErrorOutputs ErrorGeneric 
// and so on ...
```
However, option 2 assumes they have exact same set of fields so I guess former example is more flexible.

Reply:
This code I did not edit. `arc.go` is generated from `arc.yml`.